### PR TITLE
Add max-message-size processing

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -2514,7 +2514,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>An "a=sctp-port" attribute MUST be present, and it MUST be
                 parsed as specified in <xref target="I-D.ietf-mmusic-sctp-sdp"/>,
                 Section 5.2, and the value stored.</t>
-              <t>TODO: max message size</t>
+              <t>If present, a single "a=max-message-size" attribute MUST
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Might want to mention that "otherwise, use the specified default."

**Comment by juberti** [comment]
lg with comment

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: typo in otherwise :-)
(also, suggest comma after Otherwise)

**Comment by fluffy** [comment]
LGTM

**Comment by juberti** [comment]
lgtm

**Pull request merged**